data/recipes.yaml

```diff
@@ -478,6 +478,15 @@
   out:
   - [1, strange loop]
 
+- in:
+  - [2, copper pipe]
+  - [4, copper wire]
+  - [1, paper]
+  - [1, lodestone]
+  out:
+  - [2, hearing aid]
```

Comment:
Haha, I like that the recipe produces one for each ear :smile:

Reply:
Thanks, that was the idea. :ear_with_hearing_aid: